src/witan/app/schema.clj

```diff
@@ -114,6 +114,13 @@
    (s/required-key :forecast-id) IdType
    (s/required-key :forecast-version) s/Int})
 
+(def NewForecast
+  "Schema for creating a new forecast"
+  {(s/required-key :name)              s/Str
+   (s/required-key :model-id)         (s/either IdType s/Str)
+   (s/optional-key :description)       s/Str
+   (s/optional-key :model-properties) [ModelPropertyValue]})
```

Comment:
it feels to me like you've done a fair bit of work here that won't agree w/ align-clj-let or cljfmt. Is that the case?